### DISCUSSION TITLE
Selected share options

### DISCUSSION THREAD

**Discussion by PerpSearch** [benjaminjonard/koillection#1094] [Q&A] [answered]
While I've found little to wish for, I did think of something that would like to see.  When creating or populating an item in the collection it would be neat to have check box by each field for share/hide if the collection or item is shared.  In my case, I thought it useful to have several descriptive fields including the price that I paid for an item, but I don't want that price visible to the person(s) with whom I shared the item or collection.

**Comment by benjaminjonard** [comment, marked as answer]
It's not the first time someone asks for this, I'll give it a try for the next release. I will probably reuse the same visibility system as items (private/internal/public)

**Comment by PerpSearch** [comment]
Thanks. I thought I searched the discussions pretty well but I guess I missed that one. I look forward to seeing what pops. Thanks again.
